samples/streams/atLeast/worker.js

```diff
@@ -0,0 +1,51 @@
+addEventListener('fetch', event => {
+  event.respondWith(handle(event.request));
+});
+
+async function handle(request) {
+
+  const chunks = [
+    "never gonna give you up",
+    "never gonna let you go",
```

Comment:
Nit: misheard lyrics
```suggestion
    "never gonna let you down",
```